capsules/src/usb/usbc_client_ctrl.rs

```diff
@@ -22,58 +34,60 @@ use core::cmp::min;
 use kernel::hil;
 use kernel::hil::usb::TransferType;
 
-const DESCRIPTOR_BUFLEN: usize = 64;
+const DESCRIPTOR_BUFLEN: usize = 128;
 
 const N_ENDPOINTS: usize = 3;
 
-pub struct ClientCtrl<'a, 'b, C: 'a> {
-    // The hardware controller
-    controller: &'a C,
+/// Handler for USB control endpoint requests.
+pub struct ClientCtrl<'a, 'b, U: 'a> {
+    /// The USB hardware controller.
+    controller: &'a U,
```

Comment:
What's the rationale for renaming the controller type from `C` to `U`? I don't see any "u" in "controller".

Reply:
It can't be just _any_ controller, but has to be a USB controller. This mirrors other capsules where the types match the actual resource (A for Alarm, for example).